commands/debug.py

```diff
@@ -39,3 +41,15 @@
 @require_permission(TECHRAT)
 async def cmd_superping(context: Context):
     await context.reply("pong!")
+
+
+@command("debug-fact")
+@require_permission(TECHRAT)
+async def cmd_query(context: Context):
+    # assuming `database` is a instance of the DBM
+    database = DatabaseManager()
+    sql_query = sql.SQL("SELECT message from fact WHERE name=%s AND lang=%s")
+    sql_values = (context.words[1], context.words[2])
+    data = await database.query(sql_query, sql_values)
+    for item in data:
+        await context.reply(str(item))
```

Comment:
I believe we agreed a command of this form would not reach production.

Reply:
That is why it is in the debug module.  It will be needed for further work with the Fact Manager.

Reply:
This will be handled in a future pass.